### PR TITLE
Update RELEASE_NOTES

### PR DESCRIPTION
**Pull request by chefsalim**
Adding some updates to the 12.6.0 release notes.

@btm @lamont-granquist @mwrock @smurawski @chef/client-core

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -34,4 +34,23 @@ not designed to be a general solution to performance issues inside of chef-clien
 
 This debugging feature will mostly be useful to people who are already Ruby experts.
 
+### New ksh resource
 
+ksh is now available to use as a resource and is available as an interpreter in the script resource.
```
Comment (btm) [inline RELEASE_NOTES.md]: I might lead with what ksh is, e.g. "Korn Shell scripts can now be run using the `ksh` resource or by setting the interpreter parameter of the script resource to ksh."

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -34,4 +34,23 @@ not designed to be a general solution to performance issues inside of chef-clien
...
+
+Please see the DOC_CHANGES for more details.
```
Comment (btm) [inline RELEASE_NOTES.md]: The DOC_CHANGES.md doesn't appear to have more details and just points you here: https://docs.chef.io/release/12-6/resource_ksh.html

might as well just link there instead.

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -34,4 +34,23 @@ not designed to be a general solution to performance issues inside of chef-clien
...
+
+## Better handling of Windows service logging
```
Comment (btm) [inline RELEASE_NOTES.md]: can we be more clear that this is all about the chef client service, not necessarily a service managed with `windows_service` ?

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -34,4 +34,23 @@ not designed to be a general solution to performance issues inside of chef-clien
...
+## Dsc_resource changes and fixes
+
+* A fix was made for the Nov 2015 update of Windows 10, where the dsc_resource did not properly show the command output when converging the resource.
+* Dsc_resource could in some cases show the plaintext password when #inspected - this is now prevented from happening.
+* Previously, Chef required the LCM Refreshmode to be set to Disabled when utilizing dsc_resource. Microsoft has relaxed this requirement in WFM 5 (PowerShell 5.0.10586.0 or later).  Now, we only require the RefreshMode to be disabled when running on earlier versions of PowerShell.
+* Added a reboot_action attribute to dsc_resource. If the DSC resource indicates that it requires a reboot, reboot_action can use the reboot resource to either reboot immediately (:reboot_now) or queue a reboot (:request_reboot).  The default value of reboot_action is :nothing.
+
+## Other items
+
+There are a large number of other PRs in this release. Please eee the CHANGELOG for the full set of changes.
```
Comment (btm) [inline RELEASE_NOTES.md]: *see

File: RELEASE_NOTES.md (hunk 1 above)
Comment (chefsalim) [inline RELEASE_NOTES.md]: Done

File: RELEASE_NOTES.md (hunk 2 above)
Comment (chefsalim) [inline RELEASE_NOTES.md]: Done

File: RELEASE_NOTES.md (hunk 3 above)
Comment (chefsalim) [inline RELEASE_NOTES.md]: Done

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -34,4 +34,23 @@ not designed to be a general solution to performance issues inside of chef-clien
...
+
+## Dsc_resource changes and fixes
+
+* A fix was made for the Nov 2015 update of Windows 10, where the dsc_resource did not properly show the command output when converging the resource.
+* Dsc_resource could in some cases show the plaintext password when #inspected - this is now prevented from happening.
+* Previously, Chef required the LCM Refreshmode to be set to Disabled when utilizing dsc_resource. Microsoft has relaxed this requirement in WFM 5 (PowerShell 5.0.10586.0 or later).  Now, we only require the RefreshMode to be disabled when running on earlier versions of PowerShell.
```
Comment (smurawski) [inline RELEASE_NOTES.md]: I'd call out the full name - Windows Management Framework 5 (WMF5).  And you've got the F and M swapped.

Also, I'd say for the last sentence on this line - 
Now, we only require the RefreshMode to be disabled when running on earlier versions of PowerShell 5.

Comment (chefsalim) [inline RELEASE_NOTES.md]: Good catch - thanks, will update.

**Comment by smurawski** [comment]
:+1:

**Pull request merged**